src/wp-includes/blocks/template-part.php

```diff
@@ -0,0 +1,155 @@
+<?php
```

Comment:
Ideally this file shouldn't be touched manually on Core. That said, I had to modify calls to functions referring `gutenberg_` variations (I also renamed some functions to clarify their purpose). We should find a way to bring these changes back to Gutenberg and make sure the automatic renaming works properly.

Reply:
TBH I wasn't quite sure what the process is to bring a new block from GB into Core. Do we have docs on this? (Also see my remaining TODO, "Add minified CSS to Template Part block.")

Reply:
These come automatically when you do `npm install`. I think the first time you add a dynamic block though, you might have to whitelist it somewhere though.

Reply:
I thought I did that here 🤔 
```suggestion
<?phphttps://github.com/WordPress/wordpress-develop/pull/1796/files#diff-777759e98c224019a699e8294c23bccc34453c91b873a12cb78a828487cd5f15R56
```

> Ideally this file shouldn't be touched manually on Core. That said, I had to modify calls to functions referring gutenberg_ variations (I also renamed some functions to clarify their purpose). We should find a way to bring these changes back to Gutenberg and make sure the automatic renaming works properly.

How would we do this? Right now, the build step prefixes a few functions in this file with `gutenberg_`, which causes both the frontend and backend to fail 🤔  It'd be great if we could somehow unblock this...

cc/ @ntsekouras in case you wanna look into this 😄

Reply:
We need to update the code in the Gutenberg plugin to avoid relying on prefixed functions. The build step prefixes for the plugin but the file stay as (no prefix) for Core. It will require a package release though to make it back to here.

Reply:
>We need to update the code in the Gutenberg plugin to avoid relying on prefixed functions

I can try that. I'll share a PR to verify if my approach is correct.